client/src/components/Landing/ToolLanding.vue

```diff
@@ -1,5 +1,66 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import { BAlert } from "bootstrap-vue";
+import { storeToRefs } from "pinia";
+import { computed } from "vue";
+
+import type { FetchTargets } from "@/api/tools";
+import { useToolLandingStore } from "@/stores/toolLandingStore";
+
+import FetchLanding from "./FetchLanding.vue";
+import LoadingSpan from "@/components/LoadingSpan.vue";
+
+interface Props {
+    uuid: string;
+    secret?: string;
+    public?: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+    secret: undefined,
+    public: false,
+});
+
+const store = useToolLandingStore();
+const { claimTool } = store;
+const { claimState } = storeToRefs(store);
+
+// Start claim immediately
+claimTool(props.uuid, props.public, props.secret).then(() => {
+    console.log("tool request claimed....");
+});
```

Comment:
```suggestion
claimTool(props.uuid, props.public, props.secret).then(() => {
    console.debug("tool request claimed....");
});
```

if you do want to keep it, otherwise I'd drop it